Automattic/wp-calypso
# Pattern Library: Add Readymade Templates section

Fixes https://github.com/Automattic/dotcom-forge/issues/7811
Requires D153144-code

## Proposed Changes

Introduces a new Pattern Library section to highlight Readymade Templates:

<img width="1459" alt="Screenshot 2024-06-24 at 18 54 10" src="https://github.com/Automattic/wp-calypso/assets/1233880/d816733b-2919-44f9-9b8a-e5e63e5d5cfc">


A few notes:
- Everything is gated behind the `readymade-templates/showcase` feature flag
- The curated layouts and copy/paste sections don't reflect the colors specified in the Figma design. This will addressed in a follow-up PR
- The description of the section does not match the one specific in the Figma design because we are not going to support Big Sky customization yet.

## Testing Instructions

- Apply D153144-code and sandbox the API
- Use the Calypso live link below
- Go to `/patterns`
- Make sure there is a new section for Readymade Templates located between the category grid and page layouts
- Make sure it displays the current 4 examples
- Make sure the responsiveness is good across different viewports
- Make sure the "Copy paste" section has been moved after page layouts, and it has a light color scheme
- Check that changes are gated behind the `readymade-templates/showcase` feature flag

## Pre-merge Checklist
